app/models/offer.rb

```diff
@@ -30,8 +30,8 @@ class Offer < ActiveRecord::Base
   }
 
   scope :review_by, ->(reviewer_id){ where('reviewed_by_id = ?', reviewer_id) }
+  scope :donated_by, ->(donor_id){ where('created_by_id = ?', donor_id) }
```

Comment:
Space missing to the left of {.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.